src/libsyncengine/propagation/executor/executorworker.cpp

```diff
@@ -1073,6 +1074,34 @@ bool ExecutorWorker::checkLiteSyncInfoForEdit(SyncOpPtr syncOp, SyncPath &absolu
                                             << Utility::formatSyncPath(absolutePath).c_str() << L" (" << Utility::s2ws(id).c_str()
                                             << L")");
             ignoreItem = true;
+
+            // Update last modification date in order to avoid generating more EDIT operations.
```

Comment:
**Suggestion**: Encapsulate this addition within a method whose docstring would document the issue that is being fixed.

Are some software suspected to be responsible for those spurious EDIT operations?

Reply:
Done.
No, we do not have clue yet on how those modification dates have changed.